lib/tests/2wp.js

```diff
@@ -852,6 +852,56 @@ const execute = (description, getRskHost) => {
 
     });
 
+    it('should do a pegout and round down the weis to satoshis as expected', async () => {
+
+      // Arrange
+
+      // Create a pegin for the sender to ensure there is enough funds to pegout and because this is the natural process
+      const senderRecipientInfo = await createSenderRecipientInfo(rskTxHelper, btcTxHelper);
+      const peginValueInSatoshis = btcToSatoshis(0.5);
+      const btcPeginTxHash = await sendPegin(rskTxHelper, btcTxHelper, senderRecipientInfo.btcSenderAddressInfo, satoshisToBtc(peginValueInSatoshis));
+      await ensurePeginIsRegistered(rskTxHelper, btcPeginTxHash);
+
+      const initial2wpBalances = await get2wpBalances(rskTxHelper, btcTxHelper);
+      const initialSenderAddressBalanceInSatoshis = await getBtcAddressBalanceInSatoshis(btcTxHelper, senderRecipientInfo.btcSenderAddressInfo.address);
+      const pegoutValueInWeisBN = new BN('41234567891234560'); // 0.04123456789123456 in RBTC
+      const expectedPegoutValueInSatoshis = 4123456; // 0.04123456 in BTC
```

Comment:
Can't we use the unit converter library to somehow make this clearer?

Reply:
It turns out, we cant. The unit converter rounds differently than the Bridge. The unit converter would yield `4123457`, and the Bridge `4123456`. So, I had to hardcode it.

Reply:
Should we update the unit converter library then?